### PR TITLE
Adds license to side-service.js & modifies pom.xml to exclude node/**…

### PR DESCRIPTION
**Pull request by ajainarayanan**
… under cdap-ui folder

### CODE REVIEW

**Comment by elfenheart** [comment]
JS files :+1: LGTM

File: pom.xml [hunk 1]
```diff
@@ -1297,6 +1297,7 @@
                   <exclude>**/*.svg</exclude>
                   <exclude>**/bower_components/**</exclude>
                   <exclude>**/node_modules/**</exclude>
+                  <exclude>**/node/**</exclude>
```
Comment (chtyim) [inline pom.xml]: shouldn't be at top level pom. Better have it localized in the cdap-ui module (in fact not just `**/node/**`, but also the bower and node_modules one).

File: cdap-ui/pom.xml [hunk 2]
```diff
@@ -97,6 +97,36 @@
...
+              <configuration>
+                <excludes>
+                  <exclude>**/*.editorconfig</exclude>
+                  <exclude>**/*.jshintignore</exclude>
+                  <exclude>**/*.jshintrc</exclude>
+                  <exclude>**/*.svg</exclude>
+                  <exclude>**/bower_components/**</exclude>
+                  <exclude>**/node_modules/**</exclude>
+                  <exclude>**/node/**</exclude>
+                  <exclude>**/*.json</exclude>
```
Comment (awholegunch) [inline cdap-ui/pom.xml]: Add ``README.*`` to the excludes?

Comment (chtyim) [inline cdap-ui/pom.xml]: Do we have README inside cdap-ui module?

Comment (elfenheart) [inline cdap-ui/pom.xml]: we do have README in cdap-ui folder

**Comment by bdmogal** [comment]
Yes, there is a README. But like other READMEs, I added a license header at the end of it.

File: cdap-ui/pom.xml (hunk 2 above)
Comment (bdmogal) [inline cdap-ui/pom.xml]: Yes, there is a README. But like other READMEs, I added a license header at the end of it.

**Comment by bdmogal** [comment]
tested locally, I'm going to merge this, since its causing feature branches to fail RAT checks.

**Comment by anwar6953** [comment]
LGTM

**Pull request merged**